config/optional/core.entity_view_display.group_content.group_content_type_1a9b2b47fec9e.default.yml

```diff
@@ -0,0 +1,14 @@
+uuid: 97803c07-9521-443e-90a7-786ae3ff1cd9
+langcode: en
+status: true
+dependencies:
+  config:
+    - group.content_type.group_content_type_1a9b2b47fec9e
```

Comment:
I'm not sure we can do anyting here, but I'd much prefer if these said `group.content_type.group_content_type_node_page` instead of the hash at the end.

Reply:
I've tried renaming these, but then I hit a strange Field UI error, so it doesn't look as easy as just renaming the IDs and the files.